docs/developer/extending/apps/architecture/requirements/required-endpoints.mdx

```diff
@@ -0,0 +1,39 @@
+---
+title: Required endpoints
+slug: required-endpoints
+---
+
+Saleor needs two endpoints that your app must expose
+
+Manifest URL - to get your app's metadata
+Register URL - to exchange App Token
+
+# Manifest URL
+
+- Method: `GET`
+- Response - `JSON` with valid [App Manifest](https://docs.saleor.io/docs/3.x/developer/extending/apps/manifest)
+- Suggested path - `/api/manifest`
+
+Saleor will call this endpoint to get your app metadata, like name and description, but first of all - to register webhooks, permissions, and extensions.
+
+See [Manifest docs](https://docs.saleor.io/docs/3.x/developer/extending/apps/manifest) or check its [typings](https://github.com/saleor/saleor-app-sdk/blob/main/src/types.ts#L67).
+
+If your app is built with [Next.js](https://nextjs.org/), you can use a [helper from saleor/app-sdk](https://github.com/saleor/saleor-app-sdk/blob/main/docs/api-handlers.md#manifest-handler-factory), that creates a handler with minimal config.
+
+Dashboard and Saleor API will use this endpoint to install the app.
+
+# Register URL
+
+- Method: `POST`
+- Response - status `200`
```

Comment:
@krzysztofwolski SDK returns also {success: true}, is this checked by dashboard/saleor?

Reply:
https://docs.saleor.io/docs/3.x/developer/extending/apps/installing-apps 
According to docs, only the status is required. The success message seems to be an old convention.

To be sure checked registration using handler:
```typescript
import { NextApiHandler, NextApiResponse } from "next";

export const handler: NextApiHandler = (req, res) => {
  res.status(200);
  return;
};

export default handler;

```

And API marked app as registered